CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+Want to contribute? Great! There are many ways to give back to the project, whether it be writing new code, fixing bugs, or just reporting errors. All forms of contribution are encouraged!
+
+## Reporting issues
+
+Notice something amiss? Have an idea for a new feature? Feel free to to write an [Issue](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/about-issues) on the GitHub repository about anything that you feel could be fixed or improved. Examples include:
```

Comment:
I'm not sure about this, I'm just sharing a thought:

Should we link to the "new issue" link (https://github.com/NebulousLabs/skynet-js/issues/new) directly instead of (or in addition to) the current link which only explains what an issue is?

Pro: It's easier for people to create issues right from this doc.

Con: It's easier for people to create issues right from this doc, without putting any actual thought in them. :)

Reply:
That would be tricky to do from a generic doc. To clarify, once approved this PR will be closed and the doc will be added as an organization-wide default.

I thought I would link to the official docs on Issues as it also explains how to open an issue. Maybe not ideal.

When this doc goes live for the organization I'll play around and see if there's a way to point to the issues for each repo using relative links.

Reply:
OK, that makes sense.